P920.tex

```diff
@@ -0,0 +1,154 @@
+\texttt{EXPTIME},
+\texttt{CALIB}.
+
+Butler itself has native support for region and time queries.
+Support for
+\texttt{ID},
+\texttt{TARGET},
+\texttt{FACILITY},
+and \texttt{COLLECTION} are coming soon.
+For the \texttt{FACILITY} keyword we intend to match the AAS Facility names of ``Rubin:Simonyi'' for the Simonyi Survey Telescope and ``Rubin:1.2m'' for the Rubin Auxiliary Telescope.
+The Butler does not have a concept of facility but the correct value will be derived from the instrument name.
+Butler does have a concept of target but in general the value of the field is not of interest since it will usually refer to a field name generated by the survey scheduler.
+
+\section{Model Mismatches}
+
+The Butler data model as used by Rubin Observatory does result in some implementation difficulties.
+
+\subsection{Instrument for coadds}
+
+Currently, as defined by the Rubin Science Pipelines team, the Butler registry does not have an instrument associated with co-adds.
+The instrument is recorded in the datasets and in the collection names but when the pipelines were initially defined it was deemed to be unimportant to include the instrument information in the dataset type definition itself.
+This causes difficulties in SIAv2 in a Butler repository where LATISS and LSSTCam data are both available since there is no way to know from the query system which is which.
+Currently it is deemed to be too disruptive to modify the pipeline connections to propagate the instrument into the co-adds Butler dataset definition.
+Once we implement full provenance tracking it should be possible to determine the instrument using the provenance to determine the original datasets that went into the co-add.
+
+\subsection{Exposure time for co-adds}
+
+Individual observations do include exposure times in the Butler registry metadata but this information is not available to co-adds.
+This is because the median exposure time for a co-add is a derived quantity that is not know when the Butler coordinate space is defined.
+Storing derived metadata, which is similar to calculating the seeing and then allowing a query based on seeing, is on the future development roadmap.
+
+\subsection{Observing dates for co-adds}
+
+Butler does know the observing dates of individual observations but for a co-add this information is lost.
+In the future when a full Butler provenance system is implemented it may be possible to derive the date range for co-adds, but it is not possible at this time.
+
+\subsection{Dataset Types}
+
+Butler makes extensive use of what we call ``dataset types'' which define each product type in a pipeline.
+Examples can be \texttt{visit\_image}, \texttt{difference\_image}, and \texttt{deep\_coadd}.
+Currently there is no standardized way in SIAv2 for a query to specify these.
+We are looking at adding an extension, potentially using a \texttt{DPSUBTYPE} query parameter that will map directly to the Butler dataset type, possibly with an `lsst` prefix.
```

Comment:
`lsst` at the end got typeset with, literally, two backticks, which looks weird in the render.  Maybe single straight quotes are what you wanted here?  Or maybe `\texttt{lsst}` ?